docs/lcb/notes/feature-android_listeners.md

```diff
@@ -0,0 +1,65 @@
+# LiveCode Builder Language
+## Android Listener support
+An binding string variant has been added which allows the user to 
+create an interface proxy - that is an instance of a generic 
+Proxy class for a given interface. 
+
+This effectively allows LCB handlers to be registered as the targets 
+for java interface callbacks, such as event listeners.
+
+The syntax is as follows:
+
+	foreign handler _JNI_CreateListener(in pMapping) returns JObject binds to "java:listener.class.path>interface()"
+
+The foreign handler binding to such a function takes a value that should
+either be a `Handler` or an `Array` - if it is a `Handler`, the specified 
+listener should only have one available callback. If the listener has 
+multiple callbacks, an array can be used to assign handlers to each. Each 
```

Comment:
Do we need a note about 'overloaded' methods in the interface? I suspect it is a rare-case (as it would be horrendously bad design), but we should perhaps flag it and say it isn't currently supported (unless it is...).